### ISSUE TITLE
Home Page - Missing Text for DOL.gov Link

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#7]
While testing using the DHS Trusted Tester process, I found there is no text for the www.dol.gov link that is hidden behind the graphic of workers. 

Section 508 - 22(a)
WCAG 2.0 - 2.2 Web: Images

**Comment by jfc3-dol** [comment]
Here is a screenshot showing the four agency logos/seals missing ALT text.

![worker dot gov - agency logos or seals missing alt text](https://cloud.githubusercontent.com/assets/10423487/19775243/d1b721ac-9c3d-11e6-8c66-c7fef2fca7fe.png)

**Comment by adamsfallen** [comment]
Updated - aebbbf8ea8fa988017f8c2cedf8f70ad5a3a2a75

**Comment by jfc3-dol** [comment]
Those changes look good to me, since there is no need to DOL home page with that link.
